communities.json

```diff
@@ -292,6 +292,13 @@
       "inviteCode": "BuildCraft",
       "githubUrl": "https://github.com/BuildCraft/BuildCraft"
     },
+    {
+      "logo": "tachiyomi.svg",
```

Comment:
This should be `.png` (unless you have an SVG version of the logo, in which case we'd prefer you submit that).